ui/src/components/view/SearchView.vue

```diff
@@ -225,13 +225,13 @@ export default {
     styleSearch () {
       if (!this.searchFilters || this.searchFilters.length === 0) {
         return {
-          width: '100%',
+          width: '85%',
```

Comment:
this is probably not a good idea; maybe want to check/change size based on the type of screen; we've like desktop, tablet, mobile; or have the width as a ratio of the screen width?

Reply:
agree